session.go

```diff
@@ -684,7 +684,7 @@ func (s *session) logPacket(packet *packedPacket) {
 		return
 	}
 	if utils.Debug() {
-		utils.Debugf("-> Sending packet 0x%x (%d bytes), %s", packet.number, len(packet.raw), packet.encryptionLevel)
+		utils.Debugf("-> Sending packet 0x%x (%d bytes) for connection %x to %s, %s", packet.number, len(packet.raw), s.connectionID, s.conn.RemoteAddr().Network()+" "+s.conn.RemoteAddr().String(), packet.encryptionLevel)
```

Comment:
Can we use the same format `(s -> r)` here as for the "Starting new connection" line above?